readthedocs/search/documents.py

```diff
@@ -70,7 +70,6 @@ class PageDocument(RTDDocTypeMixin, DocType):
     # Metadata
     project = fields.KeywordField(attr='project.slug')
     version = fields.KeywordField(attr='version.slug')
-    path = fields.KeywordField(attr='processed_json.path')
```

Comment:
Why are we removing this?

Reply:
Got confused the nginx static files service...
and changed everything with the `full_path`.
Undoed this change.